Create salus.yaml

Piccirello opened the pull request.

Commit:
Use Salus security scanner

This will automatically scan for vulnerable dependencies.

bvallelunga reviewed (commented):
Please add a name to the job

ScottBrenner commented on .github/workflows/salus.yaml:
```diff
@@ -0,0 +1,19 @@
+    runs-on: ubuntu-latest
+    name: Salus Security Scan
+    steps:
+    - uses: actions/checkout@v1
```
```suggestion
    - uses: actions/checkout@v2
```
`v2` is the latest https://github.com/actions/checkout#whats-new

Piccirello replied:
Thanks for the suggestion!

Commit:
Resolve gosec errors and add ignore rules

Ignore rules:
G204: Subprocess launched with function call as argument or cmd arguments
G304: Potential file inclusion via variable
G402: TLS InsecureSkipVerify set true
